mypy.ini

```diff
@@ -1,5 +1,5 @@
 [mypy]
-files = hooks,providers
+files = hooks,providers/openfeature-provider-ofrep
```

Comment:
[curiosity] why do we add here the ofrep provider, and not the flagd provider, should we maybe add both?

Reply:
same reason as before, moving away from a central approach